docs/documentation/maintenance.md

```diff
@@ -478,8 +494,11 @@ The following directories are currently unmaintained:
 * Services/Notifications
 * Services/Object
 * Services/OnScreenChat
+* Services/OpenId
 * Services/PDFGeneration
+* Services/PEAR
 * Services/Password
+* Services/Payment
```

Comment:
Hi @shecken, looks as if you ran this on an older ILIAS version, payment should be gone...